Back/Reciplastk.Gateway/DataAccess/Shipment.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Reciplastk.Gateway.DataAccess;
+
+[Table("shipment")]
+public partial class Shipment
+{
+    [Key]
+    [Column("shipmentid")]
+    public int Shipmentid { get; set; }
+
+    [Column("customerid")]
+    public int Customerid { get; set; }
+
+    [Column("employeeid")]
+    public int Employeeid { get; set; }
+
+    [Column("shipmenttypeid")]
+    public int Shipmenttypeid { get; set; }
+
+    [Column("shipmentstartdate", TypeName = "timestamp without time zone")]
+    public DateTime Shipmentstartdate { get; set; }
+
+    [Column("shipmentstartend", TypeName = "timestamp without time zone")]
+    public DateTime Shipmentstartend { get; set; }
```

Comment:
```suggestion
    public DateTime Shipmentenddate { get; set; }
```
Fix property and table field name